CSV/boxes.csv

```diff
@@ -9,6 +9,7 @@ bxml,binary XML container,ISO
 ccid,OMA DRM Content ID,OMA DRM 2.1
 cdef,type and ordering of the components within the codestream,JPEG2000
 clip,Reserved,ISO
+clli,Content Light Level,MIAF
```

Comment:
Sorry to only notice this now...this should be in sample-entry-boxes, labeled as specific to Video